app/controllers/departments_controller.rb

```diff
@@ -0,0 +1,57 @@
+class DepartmentsController < ApplicationController
+
+
+  def index
+    @departments = Department.sorted
+    render (:index)
+  end
+
+  def show
+    @department = Department.find(params[:id])
```

Comment:
This same execution `Department.find(params[:id])` is being done on several methods. This is called redundant code. We'll try to avoid this kind of code. To do that there are several callback methods. Please have a look here to get some idea.